pyro/__init__.py

```diff
@@ -5,8 +5,10 @@
 import numbers
 import warnings
 from collections import OrderedDict
+from contextlib import contextmanager
 from inspect import isclass
 
+import pyro
 import torch
 
 import pyro.poutine as poutine
```

Comment:
nit: for consistency we should use the full namespace or not. `poutine.method()` vs `pyro.infer.method()` are both used in this file.

Reply:
no strong preference which is used, though we should agree on a convention